Views/Timeline/sourceevent.html.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+$message = $event['extra']['message'];
+$logs    = $event['extra']['logs'];
+?>
+<dl class="dl-horizontal small">
+    <dt><?php echo $view['translator']->trans('mautic.contactsource.timeline.logs.message'); ?></dt>
+    <dd><?=$message; ?></dd>
```

Comment:
We do not support short tags in this project, because it's a community project that goes way back to php 5.3 still.